### ISSUE TITLE
I found why it doesn't work!

### ISSUE THREAD

**Issue by Golbstein** [gzuidhof/nn-transfer#16]
the "view" in pytorch and "flatten" in keras work differently. 
To fix this problem add this layer before "Flatten":

`x = Lambda(lambda x: K.permute_dimensions(x, (0, 3, 1, 2)))(x)`

**Comment by gzuidhof** [comment]
Thanks! I would love to see a PR for this. Let me know if I can help

**Comment by Golbstein** [comment]
https://github.com/Golbstein/pytorch_to_keras

**Comment by rjarbour** [comment]
Great work!

Despite your solution I dont seem to get accurate results, I havent done a direct comparison yet, Ive just scanned over the results. Im going to tinker with it a bit more before I post my model and results. Im not really sure if it will be useful though since its a pretty deep model.

I can say now that the model uses Relu activation layers, Maxpool2D, Conv2d, and batchnorm.

**Comment by Golbstein** [comment]
In my project I used exactly this layers: Conv2d, MaxPool, BatchNorm and relu activation and I could get the same output

**Comment by rjarbour** [comment]
Im not really sure if Ive made a mistake, do you think MaxPool2D works properly? I'll try to step through it later to see if it produces the right output.

**Comment by Golbstein** [comment]
It worked for me
